R4ML/R/zzz.R

```diff
@@ -313,7 +313,7 @@ r4ml.session <- function(
   }
   
   # SparkR session init
-  sparkr.init <- function() {
+  sparkr.init <- function(...) {
```

Comment:
We should consider adding test cases to make sure arguments passed to `r4ml.session()` are respected by checking the output of `sparkR.conf()`